frontend/src/store/features/wallet/walletSlice.ts

```diff
@@ -1,8 +1,12 @@
-"use client"
+"use client";
 import { createSlice, createAsyncThunk, PayloadAction } from "@reduxjs/toolkit";
 import { getWalletAmino } from "../../../txns/execute";
 import { isWalletInstalled } from "./walletService";
-import { isConnected, setConnected, setWalletName } from "../../../utils/localStorage";
+import {
+  isConnected,
```

Comment:
- Let's use better name for `connectWalletV1`.
- We are not going to have `testnets` in the same domain, so let's completely remove `testnets` field.
- As we discussed previously, let's request a wallet connection for all networks in a single call, just like the code in the master branch.